doc/api/sqlite.md

```diff
@@ -634,6 +634,17 @@ added: v22.15.0
 By default, if an unknown name is encountered while binding parameters, an
 exception is thrown. This method allows unknown named parameters to be ignored.
 
+### `statement.setReturnArrays(enabled)`
+
+<!-- YAML
+added: 22.16.0
```

Comment:
Set the version based on the release reference mentioned in the PR description.

cc @RafaelGSS